jazz_codeq/config/dev-config.json

```diff
@@ -3,15 +3,15 @@
 	"SONAR_PASSWORD": "{sonar_creds}",
 	"SONAR_URL": "{sonar_url}",
 	"SONAR_PROJECT_KEY" : "{sonar_project_key}",
+	"SONAR_PROTOCOL": "http://",
 	"SONAR_ENV_SERVICE": "/api/measures/search_history",
-	"ALLOWED_METRICS": ["security", "code-coverage", "code-smells", "lines-of-code", "files", "vulnerabilities"],
+	"ALLOWED_METRICS": ["security", "code-coverage", "code-smells", "lines-of-code", "files"],
 	"METRIC_MAP": {
 		"security": "vulnerabilities",
 		"code-coverage": "coverage",
 		"code-smells": "code_smells",
 		"lines-of-code": "ncloc",
-		"files": "files",
-		"vulnerabilities": "high_severity_vulns"
+		"files": "files"

```

Comment:
also what is the deal with this metric `vulnerabilities`? @rajeevr2715 can you confirm that owasp is running in gitlab stack and is publishing data to sonar?

Reply:
@suryajak owasp is running perfectly, but "Dependency-Check" plugin was not installed in the SonarQube, so the owasp reports were not published into SonarQube. I have updated the container script and installed the plugin.